### PR TITLE
Language tweaks to sunday.tex up to line 789

### PR DESCRIPTION
**Pull request by jnicho02**
I haven't finished....got to cook dinner ;)

### CODE REVIEW

File: sunday.tex [hunk 1]
```diff
@@ -32,7 +32,7 @@ \section*{Sunday Sessions Descriptions}\label{sunday-descriptions}
   Flexible Routing with GraphHopper%
 }{%
 }{%
-  In this talk we try give an overview on how to use GraphHopper to provide a more flexible routing
+  In this talk we give an overview on how to use GraphHopper to provide more flexible routing
   (based on weather information, road class, road width,\dots) and how this could be also used for
```
Comment (boothym) [inline sunday.tex:36]: "\dots" - replace with " etc."?

File: sunday.tex [hunk 2]
```diff
@@ -49,9 +49,9 @@ \section*{Sunday Sessions Descriptions}\label{sunday-descriptions}
   "`Mapathon, mapathon, mapathon!"'%
```
Comment (boothym) [inline sunday.tex:49]: Three different quotation marks here, are they needed at all?

**Comment by jnicho02** [comment]
It appears to come out okay in the PDF so I didn't mess with it...

**Comment by boothym** [comment]
The "Our Falkirk" talk has double quotes which are fine, but this Mapathon one has extra single quotes and a double quote the wrong way round:
map

**Comment by jnicho02** [comment]
Ah yes, I see them now. I see you've added issue #15 for other stuff which is good. Try to keep issues about a single thing if possible.

The pull request does not attempt to fix everything in one go, but comment here if any of the changes it contains could be incorrect or debatable. I'm sincerely hoping that these pull requests will be merged promptly so that I/we can do a second pass.

File: sunday.tex [hunk 3]
```diff
@@ -49,9 +49,9 @@ \section*{Sunday Sessions Descriptions}\label{sunday-descriptions}
...
 }{%
 }{%
-  Who benefits from the mapathons: between (over)communication and (over)attribution, critical
+  Who benefits from mapathons?: between (over)communication and (over)attribution, critical
```
Comment (Nakaner) [inline sunday.tex]: Shouldn't "between" be written with a capital B and the colon removed?

Comment (jnicho02) [reply sunday.tex]: Indeed. Have updated accordingly.

Comment (jnicho02) [reply sunday.tex]: Well I thought I had....cannot see it having coming through to the pr yet

File: sunday.tex (hunk 2 above)
Comment (Nakaner) [reply sunday.tex:49]: Oh, sorry, I converted most quotation marks to ```` and `''` which are the commands for proper English quotation marks in LaTeX, but it seems that I forgot some.

File: sunday.tex (hunk 3 above)
Comment (Nakaner) [reply sunday.tex]: I will go through the whole booklet with an regular expression to look for lowercase after colon and check if a sentence ends at the colon.

**Comment by Nakaner** [comment]
I merge the pull request, pushed the change (hoping that GitHub recognises it) and rebuilt the PDFs on my server.

**Pull request closed without merging**